src/plugins/intel_cpu/src/nodes/llm_mlp.cpp

```diff
@@ -0,0 +1,652 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "llm_mlp.h"
+
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include "common/bfloat16.hpp"
+#include "common/cpu_memcpy.h"
+#include "cpu/x64/cpu_isa_traits.hpp"
+#include "cpu/x64/jit_generator.hpp"
+#include "shape_inference/shape_inference_internal_dyn.hpp"
+#include "utils/plain_tensor.hpp"
+
+#if defined(OPENVINO_ARCH_X86_64)
+#include "kernels/x64/mlp_kernel.hpp"
+using TileConfig = ov::Extensions::Cpu::TileConfig;
+using TileConfiger = ov::Extensions::Cpu::TileConfiger;
+#endif
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+
+namespace {
+
+#if defined(OPENVINO_ARCH_X86_64)
+using namespace dnnl::impl::cpu::x64;
+
+class AutoTileConfiger : public TileConfiger {
+public:
+    AutoTileConfiger() : TileConfiger() {}
+    ~AutoTileConfiger() {
+        do_config(nullptr);
+    }
+    void* last_cfg = nullptr;
+    void do_config(void* cfg) {
+        if (cfg != last_cfg) {
+            (*this)(cfg);
+            last_cfg = cfg;
+        }
+    }
+};
+
+template <typename T>
+class Singleton {
+public:
+    static T& get() {
+        static T instance;
+        return instance;
+    }
+};
+
+static PlainTensor& getC(int ithr) {
+    static std::vector<PlainTensor> all_C(parallel_get_max_threads());
```

Comment:
The static PlainTensor may consider to be removed (in the future PR).

Reply:
Sure!